eNMS/objects/routes.py

```diff
@@ -63,6 +65,41 @@ def objects_management():
     )
 
 
+@blueprint.route('/object_download', methods=['GET'])
+@login_required
+def objects_download():
+    nodes = Node.serialize()
+    ws = {}
+    wb = xlwt.Workbook()
+    style0 = xlwt.easyxf('font: name Times New Roman, color-index black, bold on',
+                         num_format_str='#,##0.00')
+    style1 = xlwt.easyxf(num_format_str='#,##0.00')
+    header_index = 0
+    # Write tabs and headers
+    for tab, header in type_to_public_properties.items():
+        column = 0
+        ws[tab] = wb.add_sheet(tab)
+        ws[tab].row_index = 1
+        for entry in header:
+            ws[tab].write(header_index, column, entry, style0)
+            column = column + 1
+    column = 0
+    for node in nodes:
+        for k, v in node.items():
+            if k is not 'id':
+                try:
+                    ws[node['type']].write(ws[node['type']].row_index, column, v, style1)
+                    column = column + 1
+                except Exception as e:
+                    continue
+        column = 0
+        ws[node['type']].row_index = ws[node['type']].row_index + 1
+    # Done writing rows
+    obj_file = Path.cwd() / 'eNMS' / 'objects' / 'objects.xls'
```

Comment:
can you export the files in `eNMS/projects` instead of in the code ? That's where all projects are supposed to be located (https://github.com/afourmy/eNMS/tree/master/projects)